src/multipart.jl

```diff
@@ -126,12 +157,11 @@ mutable struct Multipart{T <: IO} <: IO
     data::T
     contenttype::String
     contenttransferencoding::String
-    name::String
 end
 
-function Multipart(f::Union{AbstractString, Nothing}, data::T, ct::AbstractString="", cte::AbstractString="", name::AbstractString="") where {T<:IO}
```

Comment:
`name` seems unused (?) and possibly a relic of old code where `Multipart` was not the value in `Form`?

Reply:
Yeah, that sounds right

Reply:
Oh, but we do have some tests that rely on it, which are failing here

Reply:
Yea it seems that parsing a multipart body uses the name field. Perhaps parsing should give back `Vector{Pair{name::String,Multipart}}` instead to mirror how you construct a `HTTP.Form`, but I will leave that for another day and just leave the `name` field here.